amethyst_animation/src/systems/control.rs

```diff
@@ -194,6 +194,14 @@ where
             controls.remove(entity);
         }
     }
+
+    fn setup(&mut self, res: &mut Resources) {
```

Comment:
Always call `Self::SystemData::setup`, too.

Reply:
haha @Rhuagh just mentioned that on the chat, fixed!